java/ql/lib/ext/org.apache.dubbo.rpc.cluster.router.state.model.yml

```diff
@@ -0,0 +1,6 @@
+extensions:
+  - addsTo:
+      pack: codeql/java-all
+      extensible: sinkModel
+    data:
+      - ["org.apache.dubbo.rpc.cluster.router.state", "AbstractStateRouter", True, "setUrl", "(URL)", "", "Argument[0]", "request-forgery", "ai-manual"]
```

Comment:
This looks like an internal router, i.e. the application receives a request and, depending on the URL, it gets routed to a specific part of the application. I wouldn't say this `URL` is used to do a server-side request to another server.

We should remove this file.